en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -266,78 +265,81 @@ all_data <- all_data %>% mutate(
 
-Furthermore, both Wordfish and topic modeling involve ‘unsupervised’ methods - neither rely on the user pre-coding some portion of the data before modeling. Instead, these two algorithms both look at differences between documents, identifying 'natural' groupings along a dimensional scale by comparing the frequencies of words in each document to the frequencies of those words in the other documents.  Both models identify and "weigh" more heavily words whose frequency changes a lot between documents, relying particularly heavily on these word patterns to cluster documents along a scale.
+Another important similarity between Wordfish and topic modeling is that both treat documents as ['bag of words'](https://en.wikipedia.org/wiki/Bag-of-words_model). This means that the models only look at word frequency, and ignore word order: it doesn’t matter ***where*** words occur in a document, just ***which*** words occur, and with what frequency.  
 
-Another important similarity between Wordfish and topic modeling is that both treat documents as “bags of words”.  This means that the models only look at word frequency, and ignore word order. That means that it doesn’t matter ***where*** words occur in a document, just ***which*** words occur, and how many times each of those occurs.  
+Bag-of-words modeling can be problematic for longer texts where different sections of content (paragraphs, pages, chapters) might convey different types of meaning depending on their context. But social media comments tend to be very short and usually only discuss a single idea, so the bag-of-words approach is unlikely to miss key information.
+
+A significant shared strength of both models is their ability to refine results by passing over the data multiple times. When a Wordfish model is initialized, all of the parameters it measures are a ‘first best guess’ at the latent scaling of documents and words, which gives a helpful level of general insight. Depending on the quality of the text data, these models are then able to refine their initial predictions, gradually closing in on even more statistically robust and insightful models.
+
+The key differences between Wordfish scaling and topic modeling, however, are the specific statistical approaches taken, and their most useful outputs. Topic models can generate any number of topics discussed in a corpus, whereas Wordfish always scales on a single dimension (thus limitied to two topics) – but it provides much more under-the-hood information about the contribution of each word and document to the formation of this scale.
+
+### Latent Meaning

```

Comment:
I chose to move this section below ### Document Feature Matrices (DFM), as it seemed to flow better. Let me know if this works for you.